src/dune/dune.py

```diff
@@ -101,6 +103,10 @@ def create_node(self, nod):
         print("Creating node [" + nod.name() + "]")
         self._docker.execute_cmd(['mkdir', '-p', nod.data_dir()])
 
+    def get_current_nodeos_version(self):
+        stdout, stderr, exit_code = self._docker.execute_cmd(['nodeos', '--version'])
+        return stdout[1:].split('.')
```

Comment:
```suggestion
        return stdout[1:].split('.-')
```

Reply:
Thank you, completely forgot about the occasional suffix.

Reply:
For future, we probably should split on '.-+'. Forgot about build suffixes until just now.